tests/components/utility_meter/test_sensor.py

```diff
@@ -260,49 +260,49 @@ async def _test_self_reset(hass, config, start_time, expect_reset=True):
         assert state.state == "5"
 
 
-async def test_self_reset_hourly(hass):
+async def test_self_reset_hourly(hass, legacy_patchable_time):
```

Comment:
Is this because `utility_meter` is using pattern time? We should probably remove `async_track_time_change` from it and just switch it to track_interval.

Reply:
Yes